### ISSUE TITLE
NumericTextBox should compile all attributes in AngularJS

### ISSUE THREAD

**Issue by Alex-Bubblemaster** [telerik/kendo-ui-core#4759]
### Bug report

Reported in ticket with ID 1381994 

### Reproduction of the problem

![image](https://user-images.githubusercontent.com/10980842/51121608-7efc6100-1820-11e9-88cb-a54b62b19b69.png)

[Dojo](https://dojo.telerik.com/@bubblemaster/oYojEQec)

### Current behavior

When attributes are added to the NumericTextBox element, they are copied over to the input with class "k-formatted-value". Unfortunately, they are not compiled with the scope.

### Expected/desired behavior

All attributes should be compiled

### Workaround

```
$scope.$on("kendoWidgetCreated", function(e){
  var interpolatedTitle = $interpolate($scope.numeric.element.prev().attr("title"))();
  $scope.numeric.element.prev().attr("title", interpolatedTitle);            
});
```

[Dojo](https://dojo.telerik.com/OruquLEZ)

### Environment

* **Kendo UI version:** 2018.3.1017
